app/src/main/res/values/styles.xml

```diff
@@ -37,12 +37,19 @@
 
         <!-- TODO: remove when the components can get correct custom ?attr/colorPrimary -->
         <item name="colorPrimary">?attr/colorPrimary</item>
+        <item name="android:itemBackground">@drawable/menu_pop_up</item>
+        <item name="actionBarPopupTheme">@style/SubMenuStyle</item>
```

Comment:
This looks like the right attribute to set, and we already have a style called `ToolbarPopupTheme`, so creating a new one is not necessary.